spec/classes/chrony_spec.rb

```diff
@@ -130,7 +130,7 @@
         end
 
         context 'chrony::config' do
-          case facts[:osfamily]
+          case facts['os']['family']
```

Comment:
I didn't write it everywhere since it was a repeated pattern throughout the file, but please update all instances.